@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf

```diff
@@ -34,26 +30,15 @@ if (isAutoHoverOn _heli) then {
 };
 
 //Environment
-private _altitude          = _heli getVariable "fza_sfmplus_PA"; //0;     //ft
-private _altimeter         = 29.92; //in mg
-private _temperature       = _heli getVariable "fza_sfmplus_FAT"; //15;    //deg c 
-
-private _referencePressure = _altimeter * IN_MG_TO_HPA;
-private _referenceAltitude = 0;
-private _exp               = -GRAVITY * MOLAR_MASS_OF_AIR * (_altitude - _referenceAltitude) / (UNIVERSAL_GAS_CONSTANT * (_temperature + DEG_C_TO_KELVIN));
-private _pressure          = ((_referencePressure / 0.01) * (EXP _exp)) * 0.01;
-
-private _densityAltitude   = (_altitude + ((SEA_LEVEL_PRESSURE - _altimeter) * 1000)) + (120 * (_temperature - (STANDARD_TEMP - ((_altitude / 1000) * 2))));
-private _dryAirDensity     = (_pressure / 0.01) / (287.05 * (_temperature + DEG_C_TO_KELVIN));
+[_heli] call fza_sfmplus_fnc_environment;
 
 //Velocities
-[_heli, fza_ah64_sfmplusEnableWind] call fza_sfmplus_fnc_getVelocities;
+[_heli] call fza_sfmplus_fnc_getVelocities;
 [_heli] call fza_sfmplus_fnc_getAccelerations;
```

Comment:
Could your moving averages / accelerations be improved by running under fixed?

I haven't tested or require this as a change, just an idea I had